### DISCUSSION TITLE
[Report Button Plugin] - Request: Date+Time

### DISCUSSION THREAD

**Discussion by Edeka3** [runelite/runelite#17231] [Ideas]
Hi there, I would like the Report Button plugin to show both the date and the time.

Just a simple QoL request.

Thank you!

**Comment by Felanbird** [comment]
How would this look? I don't really see a readable outcome that displays both, especially since the date barely fits on it's own.

**Reply by Edeka3** [reply]
11/27 - 9:25

**Comment by Edeka3** [comment]
![jpo0w7qkkdw11~3](https://github.com/runelite/runelite/assets/64228344/fc36901a-9b3b-431a-a98a-c905c5e29ffd)

It should be 11/27, no idea why my crappy image software made it :
